state-synchronizer/src/executor_proxy.rs

```diff
@@ -182,8 +182,11 @@ impl ExecutorProxyTrait for ExecutorProxy {
         limit: u64,
         target_version: u64,
     ) -> Result<TransactionListWithProof> {
+        let start_version = known_version
+            .checked_add(1)
+            .ok_or_else(|| format_err!("Known version too high"))?;
```

Comment:
nit: maybe "Known version has overflown"?

Reply:
it hasn't yet at this point, its value is still `u64::max`, which is a-normally high :D